Nixie/Nixie.c

```diff
@@ -925,17 +1012,22 @@ void set_nixie_timedate(uint8_t x, uint8_t y, char z)
 		
 	} //switch
 
-	nixie_bits &= 0x00FFFFFF; // Clear decimal point bits 31 to 24
-	nixie_bits8 &= 0x00;	  // Clear LEFT_DP1 to LEFT_DP4	
-	nixie_bits8 = (LEFT_DP3 | LEFT_DP5);
+	//nixie_bits &= 0x00FFFFFF; // Clear decimal point bits 31 to 24
+	nixie_bits8 &= 0x00; // Clear LEFT_DP1 t/m LEFT_DP4

```

Comment:
Overbodig dit. Zeg dan: nixie_bits8 = 0x00; Bij nixie_bits is dat wel nodig, omdat je daar niet alle bits wilt resetten.